ocaml/idl/datamodel_vm.ml

```diff
@@ -1379,16 +1379,16 @@ let set_NVRAM_EFI_variables = call ~flags:[`Session]
              Deprecated, rel_boston, "Field was never used"]
              "PCI_bus" "PCI bus path for pass-through devices";
            field  ~ty:(Map(String, String)) "other_config" "additional configuration" ~map_keys_roles:["pci", _R_POOL_ADMIN; ("folder",(_R_VM_OP));("XenCenter.CustomFields.*",(_R_VM_OP))];
-           field ~qualifier:DynamicRO ~ty:Int "domid" "domain ID (if available, -1 otherwise)";
+           field ~qualifier:StaticRO ~ty:Int ~default_value:(Some (VInt (-1L))) "domid" "domain ID (if available, -1 otherwise)";
```

Comment:
Why does the domid need to become `StaticRO` ? Suspended VMs still have domid -1 IIRC. They definitely don't have an associated domain.

Reply:
Just checked and it seems you're right. I'll try without but I remembered differently. ^^'